platform_plugin_communications/tests/test_views.py

```diff
@@ -204,3 +205,109 @@ def test_send_email_to_individual_learners_with_schedule(
             "course_id": course_id,
             "success": True,
         }
+
+
+class TestSearchUsersAPIView(TestCase):
+    """
+    Test case for search_learners_api_view.
+    """
+
+    @patch("platform_plugin_communications.api.views.User")
+    def test_search_users(self, mock_User):
+        """
+        Test case for sending email to individual learners.
+        """
+
+        request = Mock()
+        request.method = "GET"
+        request.GET = {}
+        course_id = "course-v1:edX+DemoX+Demo_Course"
+        course_key = CourseKey.from_string(course_id)
+        users = [
+            {
+                "email": "test@openedx.org",
+                "username": "test",
+                "name": "test",
+            }
+        ]
+        users_mock = []
+        for user in users:
+            user_mock = Mock()
+            user_mock.username = user["username"]
+            user_mock.email = user["email"]
+            user_mock.profile.name = user["name"]
+            users_mock.append(user_mock)
+        mock_User.objects.filter.return_value = users_mock
+        response = search_learner(request, course_id)
+
+        mock_User.objects.filter.assert_called_once_with(
+            is_active=True,
+            courseenrollment__course_id=course_key,
+            courseenrollment__is_active=True,
+        )
```

Comment:
can we also check this filter? https://github.com/eduNEXT/platform-plugin-communications/pull/4/files#diff-13fbc08c712019edc8bfb712dff48e3f4c70405f0ea8b7da2902a3d8812c31d3R142-R147

Reply:
what's the difference between this test and the next one?

Reply:
> can we also check this filter? https://github.com/eduNEXT/platform-plugin-communications/pull/4/files#diff-13fbc08c712019edc8bfb712dff48e3f4c70405f0ea8b7da2902a3d8812c31d3R142-R147

This test case doesn't trigger that behavior. The following test triggers it

Reply:
Can we update the docstring and the test name so it reflects that?